lib/getBinaryGltf.js

```diff
@@ -118,12 +118,16 @@ function getBinaryGltf(gltf, embed, embedImage) {
     // Remove extras objects before writing
     removePipelineExtras(gltf);
 
-    // Create padded binary scene string and calculate total length
+    // Create padded binary scene buffer and calculate total length
     var sceneString = JSON.stringify(gltf);
-    var sceneLength = Buffer.byteLength(sceneString);
-    sceneLength += 4 - (sceneLength % 4);
-    var padding = new Array(sceneLength + 1).join(' ');
```

Comment:
I'm not 100% sure if there was a reason for the padding to be so big before, and then get sliced down to size. Any insights?